src/index.ts

```diff
@@ -99,6 +99,7 @@ export { Scenario, State } from "./auth/state.js"
 export { AuthCancelled, AuthSucceeded, Continuation, NotAuthorised } from "./auth/state.js"
 export { InitialisationError, InitOptions } from "./init/types.js"
 
+export * as auth from "./auth/index.js"
```

Comment:
App owned auth functions are exported from the top level as `auth`.

Reply:
Renamed this to export as `account` instead of `auth` to contextualize the consumer and producer.